puma/puma.init.erb

```diff
@@ -1,18 +0,0 @@
-description "<%=user%> puma"
-
-start on (filesystem and static-network-up)
-stop on runlevel [!2345]
-
-respawn
-
-kill signal QUIT
-
-chdir <%=app_root%>
-setuid <%=user%>
-console log
-env LANG=en_AU.UTF-8
-env RACK_ENV=<%=env%>
-
-script
-  ./bin/puma
-end script
```

Comment:
So we don't need a custom upstart config for puma anymore, the standard `upstart` template will work?